src/internal/utils/getToastAnimation.ts

```diff
@@ -0,0 +1,16 @@
+export function getToastAnimation(
+  position: 'top-center' | 'top-right' | 'bottom-center' | 'bottom-right',
+) {
+  switch (position) {
+    case 'top-right':
+      return 'animate-enterRight';
+    case 'top-center':
+      return 'animate-enterDown';
+    case 'bottom-right':
+      return 'animate-enterRight';
+    case 'bottom-center':
+      return 'animate-enterUp';
+    default:
+      throw new Error('Invalid toast position');
+  }
```

Comment:
if we aren't trying to reuse this for other animations it could be a simple map in the toast component, i.e. `{ 'top-right': 'animate-enterRight', 'top-center': 'animate-enterDown', ...}`

Reply:
sounds good. i was trying to follow the `getToastPosition` pattern, but a simple const is simpler :-)

Reply:
all set